[Mod] showing whitelisted and blacklisted users

Jan200101 opened the pull request:
Added a little clode block to show which users are blacklisted or whitelisted.
I haven't decided if showing the name is enough or to show the nickname and maybe the discriminator.

Jan200101 committed:
[Mod] showing whitelisted and blacklisted users

Jan200101 commented on cogs/mod.py:
```diff
@@ -1003,7 +1003,13 @@ def check(m):
+                userlist = ', '.join(name)
+            else:
+                userlist = "No one"
+            await self.bot.say(box('Blacklisted users:\n' + userlist))
```
Unsure if what to display ontop.
My suggestions are

- Blacklisted users :
- Blacklist :

Jan200101 commented on cogs/mod.py:
```diff
@@ -1037,7 +1043,13 @@ def check(m):
+                userlist = ', '.join(name)
+            else:
+                userlist = "No one"
+            await self.bot.say(box('Whitelisted users:\n' + userlist))
```
Unsure if what to display ontop.
My suggestions are

Whitelisted users :
Whitelist :

Jan200101 commented on cogs/mod.py:
```diff
@@ -1003,7 +1003,13 @@ def check(m):
+            if name:
+                userlist = ', '.join(name)
+            else:
+                userlist = "No one"
```
Better text for when no one is blacklisted

Jan200101 commented on cogs/mod.py:
```diff
@@ -1037,7 +1043,13 @@ def check(m):
+            if name:
+                userlist = ', '.join(name)
+            else:
+                userlist = "No one"
```
Better text for when no one is whitelisted

Jan200101 reviewed (commented):
I need some feedback for the string I marked and what text to fill in these

Jan200101 committed:
[mod] pagifying the blacklist / whitelist

Jan200101 committed:
[Mod] getting all users

Jan200101 committed:
[Mod] suggestion by calebj

Twentysix26 commented:
`Client.get_user_info` is supposed to be used as a last resort. The rate limit is one request per second.

Jan200101 commented:
Adding a exception and using it if the bot isnt on any server with that user

Jan200101 committed:
[Mod] changed user fetching (description for more)
Check added if the user is already sharing a server (guild)
with Red and fetching his information from that.
If the user is not sharing a server with Red it fetches the users global
information.

Jan200101 committed:
Merge remote-tracking branch 'upstream/develop' into develop

Jan200101 commented:
Added if statement instead of except

Twentysix26 commented:
Adding it in #701
